docs/release-notes/2708-reduce-single-trial-exp-page-loading.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**BUG FIXES**
+
+-  WebUI: Avoid waiting for an extra polling cycle to load trial data
```

Comment:
was this present in the previous release? I feel like it's new but I haven't checked. If it wasn't we don't need this.